src/components/carousel/index.tsx

```diff
@@ -455,23 +456,23 @@ class Carousel extends Component<CarouselProps, CarouselState> {
 
   renderCounter() {
     const {pageWidth, showCounter, counterTextStyle} = this.props;
-    const {currentStandingPage} = this.state;
+    const {currentPage} = this.state;
     const pagesCount = presenter.getChildrenLength(this.props);
 
     if (showCounter && !pageWidth) {
       return (
         <View center style={styles.counter}>
           <Text grey80 text90 style={[{fontWeight: 'bold'}, counterTextStyle]}>
-            {currentStandingPage + 1}/{pagesCount}
+            {currentPage + 1}/{pagesCount}
           </Text>
         </View>
       );
     }
   }
 
   renderAccessibleLayout() {
-    const {containerStyle, children, testID} = this.props;
-
+    const {containerStyle, children: propsChildren, testID} = this.props;
+    const children = Constants.isRTL && Constants.isAndroid ? React.Children.toArray(propsChildren) : propsChildren;
```

Comment:
What is this for?

Reply:
For some on RTL android sends the children reversed.